src/tests/test_kube_api.py

```diff
@@ -198,7 +198,7 @@ def capi_test(
                     agent_namespace=spoke_namespace,
                     provider_image=os.environ.get("PROVIDER_IMAGE", ""),
                     hypershift_cpo_image=os.environ.get("HYPERSHIFT_IMAGE", ""),
-                    release_image=os.environ.get("OPENSHIFT_INSTALL_RELEASE_IMAGE", ""),
+                    release_image=os.environ.get("ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE", ""),
```

Comment:
Are you sure it's being defined for test-infra based jobs?

Reply:
Hmmm, you are right, it seems that it's being passed only for operator workflows.